### ISSUE TITLE
Fix hyphenated module names

### ISSUE THREAD

**Issue by MatthewDaggitt** [agda/agda-stdlib#265]
A few modules have some non-standard names with hyphens in:
```
Algebra.Monoid-solver
Algebra.RingSolver.Natural-coefficients
Data.Colist.Infinite-merge
```
These should all have the hyphens removed (taking advantage of release 1.0 non-backwards compatibleness).

**Comment by gallais** [comment]
Should this be Milestone 1.0 rather than 0.16?

**Comment by MatthewDaggitt** [comment]
Yup, you're right. Created it now.

**Comment by MatthewDaggitt** [comment]
Closed with f0cec64
